### DISCUSSION TITLE
Add an additional resources section

### DISCUSSION THREAD

**Discussion by aklin2** [sandbox-bio/sandbox.bio#13] [Q&A] [answered]
Learning all these tools was not only incredibly useful, but quite fun as well 😄 

It would be great if there was a section that expanded on how to get involved in the bioinformatics community and use these tools that we learned.
Maybe a "Find your own project" section or an example of a project that the student can do?
Resources on how to get free genome references (i.e. https://www.internationalgenome.org/)

I suppose one project that might be interesting is aligning the COVID-19 viral genome to different genomes to find the most related species. Although I'm not sure `samtools`, `bcftools`, and `bowtie2` can do all that...

**Comment by robertaboukhalil** [comment, marked as answer]
> I suppose one project that might be interesting is aligning the COVID-19 viral genome to different genomes to find the most related species.

A viral amplicon sequencing tutorial was just released: https://sandbox.bio/tutorials?id=viral-amplicon 🙂
